celery/__init__.py

```diff
@@ -19,7 +19,7 @@
 )
 
 SERIES = 'Cipater'
-VERSION = version_info_t(3, 1, 25, '', '')
+VERSION = version_info_t(3, 1, 26, '', '')
```

Comment:
I think this will be confusing - it's not actually a 3.1.26 version as in original celery but our own version. On the other hand, I understand that you'll release it with some other version to be consumed by the monolith. I'm not sure how to handle well such a case.

Reply:
changed to 3.1.25-beekeeper-1 version